RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chrome.java

```diff
@@ -830,37 +825,37 @@ private void getAutofill() {
                 full_name = String.join(" ", first_name, middle_name, last_name);
             }
             bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_NAME_PERSON,
-                    NbBundle.getMessage(this.getClass(), "Chrome.parentModuleName"),
+                    RecentActivityExtracterModuleFactory.getModuleName(),
```

Comment:
Where should the NON-NLS used?  It is just with message strings or all quoted strings.  Will the code that parses for internationalization know the different between a constant vs. a string for a message.  For example if you look at the reset of that add call it has two strings in it for db table headers.  If the NON-NLS tag is not at the end of that statement is the parse code smart enough to ignore those?

What about String constants?

Reply:
I looked through the reset of the Chrome code and answered my questions.  the NON-NLS should stay there because of the strings in the rest of the call.